apps/penxle.com/src/lib/components/ColorPicker.svelte

```diff
@@ -240,13 +240,16 @@
     >
       <input
         style:background-color={hex}
-        class={css({
-          borderWidth: '1px',
-          borderColor: 'gray.200',
-          borderRadius: 'full',
-          marginRight: '12px',
-          size: '22px',
-        })}
+        class={cx(
+          'input-type-color',
+          css({
+            borderWidth: '1px',
+            borderColor: 'gray.200',
+            borderRadius: 'full',
+            marginRight: '12px',
+            size: '22px',
+          }),
```

Comment:
👍 👍  
P5: 근데 요런식으로 쓸수도 있을것같아용

```suggestion
          '&::-webkit-color-swatch-wrapper': {
            visibility: 'hidden',
          },
          '&::-moz-color-swatch':  {
            borderStyle: 'none',
          }
```